lms/envs/common.py

```diff
@@ -1351,6 +1351,9 @@ def _make_mako_template_dirs(settings):
 #   ]
 COURSES_WITH_UNSAFE_CODE = []
 
+# Cojail REST service
+CODE_JAIL_REST_SERVICE_HOST = 'http://127.0.0.1:8550'
```

Comment:
Should we add this setting in the production.py file so it can be overridden via `ENV_TOKENS`?